### PR TITLE
Copypasta'd

### PR DESCRIPTION
**Pull request by mhwatson**

### CODE REVIEW

File: src/services/packages/routes.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import express from "express";
-import { badRequestCatch, healthCheck,  getPouch, getPouchRedirect } from "./controllers";
+import { badRequestCatch, healthCheck,  getPouch, getPouchRedirect, getPouchRedirectToRoaster } from "./controllers";
 
 export const router = express.Router();
 
 router.get("/v1/pouches/:pouch_id/redirect", getPouchRedirect);
 router.get("/v1/pouches/:pouch_id", getPouch);
+router.get("/v1/pouches/:pouch_id/redirect-to-roaster", getPouchRedirectToRoaster);
```
Comment (kshahzada) [inline src/services/packages/routes.ts:8]: can we change this to "/v1/pouches/:pouch_id/roaster/redirect"

File: src/services/packages/logic.ts [hunk 2]
```diff
@@ -28,6 +28,18 @@ export const redirectToPouch = async (pouch_id: string) => {
...
 }
 
+export const redirectToRoaster = async (pouch_id: string) => {
+    const pouch_details = await read("Pouches", pouch_id);
+    if (pouch_details === undefined || pouch_details.Coffee_Link === undefined){
+        return resourceNotFoundError();
+    }
+    const response: ILogicResponse = {
+        // References are always arrays for some reason 🤷‍♂️
+        redirectURL: pouch_details.Coffee_Link[0],
```
Comment (kshahzada) [inline src/services/packages/logic.ts:38]: yeah for some reason, even if you mark "don't allow link to many" it still does

**Review by kshahzada — CHANGES_REQUESTED** [review]

File: src/services/packages/routes.ts (hunk 1 above)
Comment (mhwatson) [reply src/services/packages/routes.ts:8]: That's not very restful of you...

Comment (mhwatson) [reply src/services/packages/routes.ts]: Addressed!

**Review by kshahzada — APPROVED** [review]

**Pull request merged**